app/const.go

```diff
@@ -0,0 +1,34 @@
+package gaia
+
+// nolint: gosec
+const (
+	appName     = "GaiaApp"
+	upgradeName = "v7-Theta"
```

Comment:
i think we can just cheat and call this agoric-upgrade-7

Reply:
And how about `appName`?

Reply:
should stay the same - at least it's that way in agoric-upgrade-5/6

Reply:
Done.